src/ecrc-frontend/src/components/page/informationreview/InformationReview.js

```diff
@@ -100,9 +100,25 @@ export default function InformationReview({
 
           setShareAvailable(true);
         })
-        .catch(() => {
-          // If error = no share, do nothing
-          // else if other error, go to error page and display correct error
+        .catch(error => {
+          if (
+            error &&
+            error.response &&
+            error.response.status &&
+            error.response.status !== 400
+          ) {
+            setToError(true);
```

Comment:
so we dont want to go to error page if error status is 400?

Reply:
Yes. 400 is what we get back from checkApplicantForPrevCRC when it does not find anything, so it is an expected error.